// bootstrap-vue-next/dist/src/components/index.mjs
import { _, a } from "../../BAccordionItem.vue_vue_type_script_setup_true_lang-qX-1Wfby.mjs";
import { B } from "../../BAlert-0UDGGiWI.mjs";
import { _ as _2, a as a2 } from "../../BAvatarGroup.vue_vue_type_script_setup_true_lang-uRDCerzk.mjs";
import { _ as _3 } from "../../BBadge.vue_vue_type_script_setup_true_lang-IIZ8QpjG.mjs";
import { _ as _4, a as a3 } from "../../BBreadcrumb.vue_vue_type_script_setup_true_lang-nd2E3QjW.mjs";
import { _ as _5 } from "../../BButton.vue_vue_type_script_setup_true_lang-BC0Lfmfc.mjs";
import { _ as _6, a as a4 } from "../../BButtonToolbar.vue_vue_type_script_setup_true_lang-Cw3YzCpt.mjs";
import { _ as _7 } from "../../BCloseButton.vue_vue_type_script_setup_true_lang-B0EXlL9P.mjs";
import { _ as _8, a as a5, b, c, d, e, f } from "../../BCard.vue_vue_type_script_setup_true_lang-BrZMYuL-.mjs";
import { _ as _9, a as a6 } from "../../BCardText.vue_vue_type_script_setup_true_lang-Be6CD36N.mjs";
import { _ as _10, a as a7 } from "../../BCarouselSlide.vue_vue_type_script_setup_true_lang-DvJdLgu_.mjs";
import { _ as _11 } from "../../BCollapse.vue_vue_type_script_setup_true_lang-DjUDEnp7.mjs";
import { _ as _12 } from "../../BCol.vue_vue_type_script_setup_true_lang-zEDCknHc.mjs";
import { _ as _13 } from "../../BContainer.vue_vue_type_script_setup_true_lang-w2M6sC0-.mjs";
import { _ as _14 } from "../../BRow.vue_vue_type_script_setup_true_lang-69TY75-8.mjs";
import { B as B2 } from "../../BDropdown-BucS94bu.mjs";
import { _ as _15, a as a8, b as b2, c as c2, d as d2, e as e2, f as f2 } from "../../BDropdownText.vue_vue_type_script_setup_true_lang-BDYQbS3v.mjs";
import { _ as _16 } from "../../BForm.vue_vue_type_script_setup_true_lang-Crnxo4h0.mjs";
import { _ as _17, a as a9 } from "../../BFormFloatingLabel.vue_vue_type_script_setup_true_lang-BMvhcrPS.mjs";
import { _ as _18, a as a10, b as b3, c as c3 } from "../../BFormValidFeedback.vue_vue_type_script_setup_true_lang-Bs-YFQyq.mjs";
import { _ as _19, a as a11 } from "../../BFormCheckboxGroup.vue_vue_type_script_setup_true_lang-Do7WryrT.mjs";
import { _ as _20 } from "../../BFormFile.vue_vue_type_script_setup_true_lang-BfwrNM0g.mjs";
import { _ as _21 } from "../../BFormGroup.vue_vue_type_script_setup_true_lang-4-YcPJAc.mjs";
import { _ as _22, _ as _23 } from "../../BFormInput.vue_vue_type_script_setup_true_lang-KuNFoABp.mjs";
import { _ as _24, a as a12 } from "../../BFormRadioGroup.vue_vue_type_script_setup_true_lang--UYbn-2B.mjs";
import { _ as _25, a as a13 } from "../../BFormSelect.vue_vue_type_script_setup_true_lang-BigptVap.mjs";
import { _ as _26 } from "../../useFormSelect-BJO_8jcC.mjs";
import { _ as _27 } from "../../BFormSpinbutton.vue_vue_type_script_setup_true_lang-B-RgEb6R.mjs";
import { a as a14, _ as _28 } from "../../BFormTags.vue_vue_type_script_setup_true_lang-FWDxfiLi.mjs";
import { _ as _29 } from "../../BFormTextarea.vue_vue_type_script_setup_true_lang-CVNC0lcp.mjs";
import { _ as _30 } from "../../BImg.vue_vue_type_script_setup_true_lang-CoowQ3jJ.mjs";
import { _ as _31, a as a15 } from "../../BInputGroupText.vue_vue_type_script_setup_true_lang-C3oeIi3U.mjs";
import { _ as _32 } from "../../BLink.vue_vue_type_script_setup_true_lang-X9oGQtcq.mjs";
import { _ as _33, a as a16 } from "../../BListGroupItem.vue_vue_type_script_setup_true_lang-BvZYkKtO.mjs";
import { B as B3, _ as _34 } from "../../BModalOrchestrator.vue_vue_type_script_setup_true_lang-CwsLFqd0.mjs";
import { _ as _35, a as a17, b as b4, c as c4, d as d3 } from "../../BNavText.vue_vue_type_script_setup_true_lang-DE6m-KYq.mjs";
import { _ as _36, a as a18, b as b5, c as c5 } from "../../BNavbarToggle.vue_vue_type_script_setup_true_lang-CCIDsf4z.mjs";
import { B as B4 } from "../../BOffcanvas-BEhqT7oK.mjs";
import { _ as _37 } from "../../BOverlay.vue_vue_type_script_setup_true_lang-C5dzyxuB.mjs";
import { _ as _38 } from "../../BPagination.vue_vue_type_script_setup_true_lang-DH_8I7Df.mjs";
import { _ as _39, a as a19, b as b6, c as c6, d as d4 } from "../../BPlaceholderWrapper.vue_vue_type_script_setup_true_lang-CliOyeln.mjs";
import { B as B5 } from "../../floatingUi-CsxgENev.mjs";
import { _ as _40 } from "../../BPopoverOrchestrator.vue_vue_type_script_setup_true_lang-DxYnyvsD.mjs";
import { _ as _41, a as a20 } from "../../BProgress.vue_vue_type_script_setup_true_lang-BOhOz9Pk.mjs";
import { _ as _42 } from "../../BSpinner.vue_vue_type_script_setup_true_lang-BBcjZkCi.mjs";
import { _ as _43, a as a21, b as b7, c as c7, d as d5, e as e3, f as f3, g } from "../../BTable.vue_vue_type_script_setup_true_lang-CiThwOJO.mjs";
import { _ as _44 } from "../../BTableSimple.vue_vue_type_script_setup_true_lang-Dbrg2rgZ.mjs";
import { _ as _45, a as a22 } from "../../BTabs.vue_vue_type_script_setup_true_lang-AmprSk-s.mjs";
import { B as B6, _ as _46 } from "../../BToastOrchestrator.vue_vue_type_style_index_0_lang-CgGO-ddH.mjs";
import { _ as _47 } from "../../BTooltip.vue_vue_type_script_setup_true_lang-B1N8oaI-.mjs";
export {
  _ as BAccordion,
  a as BAccordionItem,
  B as BAlert,
  _2 as BAvatar,
  a2 as BAvatarGroup,
  _3 as BBadge,
  _4 as BBreadcrumb,
  a3 as BBreadcrumbItem,
  _5 as BButton,
  _6 as BButtonGroup,
  a4 as BButtonToolbar,
  _8 as BCard,
  a5 as BCardBody,
  b as BCardFooter,
  _9 as BCardGroup,
  c as BCardHeader,
  d as BCardImg,
  e as BCardSubtitle,
  a6 as BCardText,
  f as BCardTitle,
  _10 as BCarousel,
  a7 as BCarouselSlide,
  _7 as BCloseButton,
  _12 as BCol,
  _11 as BCollapse,
  _13 as BContainer,
  B2 as BDropdown,
  _15 as BDropdownDivider,
  a8 as BDropdownForm,
  b2 as BDropdownGroup,
  c2 as BDropdownHeader,
  d2 as BDropdownItem,
  e2 as BDropdownItemButton,
  f2 as BDropdownText,
  _16 as BForm,
  _19 as BFormCheckbox,
  a11 as BFormCheckboxGroup,
  _17 as BFormDatalist,
  _20 as BFormFile,
  a9 as BFormFloatingLabel,
  _21 as BFormGroup,
  _22 as BFormInput,
  _18 as BFormInvalidFeedback,
  _24 as BFormRadio,
  a12 as BFormRadioGroup,
  a10 as BFormRow,
  _25 as BFormSelect,
  _26 as BFormSelectOption,
  a13 as BFormSelectOptionGroup,
  _27 as BFormSpinbutton,
  a14 as BFormTag,
  _28 as BFormTags,
  b3 as BFormText,
  _29 as BFormTextarea,
  c3 as BFormValidFeedback,
  _30 as BImg,
  _23 as BInput,
  _31 as BInputGroup,
  a15 as BInputGroupText,
  _32 as BLink,
  _33 as BListGroup,
  a16 as BListGroupItem,
  B3 as BModal,
  _34 as BModalOrchestrator,
  _35 as BNav,
  a17 as BNavForm,
  b4 as BNavItem,
  c4 as BNavItemDropdown,
  d3 as BNavText,
  _36 as BNavbar,
  a18 as BNavbarBrand,
  b5 as BNavbarNav,
  c5 as BNavbarToggle,
  B4 as BOffcanvas,
  _37 as BOverlay,
  _38 as BPagination,
  _39 as BPlaceholder,
  a19 as BPlaceholderButton,
  b6 as BPlaceholderCard,
  c6 as BPlaceholderTable,
  d4 as BPlaceholderWrapper,
  B5 as BPopover,
  _40 as BPopoverOrchestrator,
  _41 as BProgress,
  a20 as BProgressBar,
  _14 as BRow,
  _42 as BSpinner,
  _45 as BTab,
  _43 as BTable,
  a21 as BTableLite,
  _44 as BTableSimple,
  a22 as BTabs,
  b7 as BTbody,
  c7 as BTd,
  d5 as BTfoot,
  e3 as BTh,
  f3 as BThead,
  B6 as BToast,
  _46 as BToastOrchestrator,
  _47 as BTooltip,
  g as BTr
};
